### DISCUSSION TITLE
Revert New UI

### DISCUSSION THREAD

**Discussion by mo-g** [community/community#101790] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This may be the worst user interface change I've seen since Facebook c. 2009. Please revert.

I do not care about what is happening in other peoples bloody repositories. Please get rid of this, or as I promised in my feedback when you put this through feature preview, I will leave GitHub. I am not here to socialise, I'm here to code.

**Comment by mo-g** [comment]
![howmuchofgithubismine](https://github.com/community/community/assets/10463172/6bbd8d26-8fa1-4599-9824-26b3531c1ce6)

It's always useful to include data in a bug report, right? So here's some. This image, split into a light/green and dark/red (so it should hopefully still be obvious to colour blind people, I hope?) The small light/green section top left is how much of Github is things that are me, and the large dark/red section bottom right is how much of the screen is things that Github's pushing at me.

Invert these areas (mostly mine, small amount Github) and we're good. Like this? You're trying to steal too much of my attention and screen real-estate, and it doesn't fly.

**Reply by sergey-imprezz** [reply]
yes sir! same here... this drives me nuts... what were they thinking? really.
